runtime/gc_base/StringTable.cpp

```diff
@@ -568,6 +604,10 @@ j9gc_createJavaLangString(J9VMThread *vmThread, U_8 *data, UDATA length, UDATA s
 			}
 		}
 	}
+	bool isLatin1 = isASCII;
```

Comment:
Since this variable is used differently for < 17 and >= 17, I think it needs a better name, such as isASCIIorLatin1.

Reply:
Updated.